### PR TITLE
Fix destination path for bcsymbolmap files

### PR DESCRIPTION
**Pull request by florianbuerger**
As discussed in https://github.com/CocoaPods/CocoaPods/issues/8552

Requires https://github.com/CocoaPods/cocoapods-integration-specs/pull/220 to be merged first.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,9 +12,13 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
...
 ##### Bug Fixes
 
+* Copy `bcsymbolmap` files into correct destination to avoid invalid app archives    
+  [florianbuerger](https://github.com/florianbuerger)
+  [#8548](https://github.com/CocoaPods/CocoaPods/pull/8548)
+
 * Do not use spaces around variable assignment in generated embed framework script  
   [florianbuerger](https://github.com/florianbuerger)
-  [#8548](https://github.com/CocoaPods/CocoaPods/pull/8548) 
+  [#8558](https://github.com/CocoaPods/CocoaPods/pull/8558)
```
Comment (dnkoutso) [inline CHANGELOG.md]: this does not appear to be the right link for this change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,9 +12,13 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Copy `bcsymbolmap` files into correct destination to avoid invalid app archives    
+  [florianbuerger](https://github.com/florianbuerger)
+  [#8548](https://github.com/CocoaPods/CocoaPods/pull/8548)
```
Comment (dnkoutso) [inline CHANGELOG.md]: same here this link is incorrect.

**Comment by dnkoutso** [comment]
Thanks! Please fix `CHANGELOG` and also update the integration specs reference since it was merged.

**Comment by florianbuerger** [comment]
@dnkoutso Ups, I am sorry about that. Everything should be green now. Thanks for catching my mistake 🙏

**Pull request merged**